src/multisig.rs

```diff
@@ -21,17 +27,21 @@ use failure;
 use hex;
 use secp256k1::{PublicKey, Secp256k1};
 
+/// A standard redeem script.
 #[derive(Debug, PartialEq, Clone)]
 pub struct RedeemScript(pub(crate) Script);
 
 impl RedeemScript {
+    /// Tries to interpret a raw script as a standard redeem script and returns error
```

Comment:
Maybe use "parse" instead of "interpret" here? As I can see, there is no interpretation happens here.